### PR TITLE
Give a tip about not forgetting @ in curl command.

### PR DESCRIPTION
**Pull request by tobya**
I'm running through the tutorial and and noteing things that give me a bit of trouble.

### CODE REVIEW

**Comment by mholt** [comment]
Thanks Toby! Most people copy+paste examples/instructions. Were you typing it out manually?

**Comment by tobya** [comment]
Yes,

I remember looking at it and thinking 'I'm sure that @ is important' then
not typing it.

Toby

File: src/docs/markdown/api-tutorial.md [hunk 1]
```diff
@@ -105,6 +110,8 @@ Save the config file, then update Caddy's active configuration by running the sa
 	-d @caddy.json
 </code></pre>
 
+
+
```
Comment (mholt) [inline src/docs/markdown/api-tutorial.md]: Extra lines? Can remove these, please :)

File: src/docs/markdown/api-tutorial.md [hunk 2]
```diff
@@ -73,6 +73,11 @@ Then upload it:
 	-d @caddy.json
 </code></pre>
 
+<aside class="tip">
+	Make sure you dont forget the @ in front of your filename, this tells curl you are sending a file.
```
Comment (mholt) [inline src/docs/markdown/api-tutorial.md]:
```suggestion
	Make sure you don't forget the @ in front of your filename; this tells curl you are sending a file.
```

**Review by mholt — CHANGES_REQUESTED** [review]
Just a few matters of cleanup.

**Review by mholt — APPROVED** [review]
Thanks Toby

**Pull request merged**